globals.py

```diff
@@ -239,6 +243,15 @@ def update_logger_levels(self, context):
 				elif bpy.context.preferences.addons[__package__].preferences.logger_level == '1':
 					handler.setLevel(logging.INFO)
 
+				# if the level is ERROR
+				elif bpy.context.preferences.addons[__package__].preferences.logger_level == '2':
+					handler.setLevel(logging.ERROR)
+
+		# 3. Enable or disable all logging mode.
+		enable = bpy.context.preferences.addons[__package__].preferences.logger_level != '2'
+		LookingGlassAddon.debugging_print_pylio_logger_all = enable
+		LookingGlassAddon.debugging_print_internal_logger_all = enable
+

```

Comment:
Same as in the comment above: `LookingGlassAddon.debugging_print_pylio_logger_all` and `LookingGlassAddon.debugging_print_internal_logger_all` are meant to control the console output only.
```suggestion
```

Reply:
I was planning to set both variables to `False` by default so that users don't see any console output at all, because all logging is done in the dedicated log files. However, for future development it might be easier to add a checkbox in the preference pane. Something like `Show in Console`, which sets both variables to `True` if checked and vice versa. That way developers don't have to take care that the variables are always set to `False` before committing changes. I will integrate this in a later commit, unless you want to take care of that.

Reply:
Only thing to consider is that you may still not get the first printouts to console if it's before register happens (if that's a concern), but otherwise yeah that works. I can add a tick box in a follow on commit in this PR.

Reply:
Yes, that is fine for now. As mentioned in my previous comment, I plan to take care of that in a future release. At the moment the initialization is not well structured and needs a major refactoring. I will open an issue to keep your comment here in mind for the future work on that.